test/e2e_node/gpu_device_plugin_test.go

```diff
@@ -116,20 +116,21 @@ var _ = framework.KubeDescribe("NVIDIA GPU Device Plugin [Feature:GPUDevicePlugi
 			ginkgo.By("Checking that scheduled pods can continue to run even after we delete device plugin.")
 			ensurePodContainerRestart(f, p1.Name, p1.Name)
 			devIdRestart1 = parseLog(f, p1.Name, p1.Name, deviceIDRE)
+			framework.ExpectEqual(devIdRestart1, devId1)
 			gomega.Expect(devIdRestart1).To(gomega.Equal(devId1))
```

Comment:
I think that `gomega.Expect...` at line:120 is unnecesarry.

Reply:
done